https://issues.redhat.com/browse/ACM-12024 (#6626)
open-cluster-management/rhacm-docs: observability/manage_search.adoc

+
<1> Add the namespace where search is enabled.

> . *Technology Preview:* To enable global search to view your hub and managed cluster resources, add the `global-search-preview=true` annotation to the `search-v2-operator` resource. Run the following command:

Reviewer: This is correct, but I'd like to add something to associate this feature with Multicluster Global Hub. Actually, MCGH is a required pre-requisite.

```
When using Multicluster Global Hub, enable global search to expand the search capabilities to view data from the managed hubs and managed clusters. To enable this feature, add the `global-search-preview=true` annotation to the `search-v2-operator` resource. Run the following command:
```

Or maybe this
```
Global search expands the search capabilities when using Multicluster Global Hub to manage your environment.  To enable this feature, add the `global-search-preview=true` annotation to the `search-v2-operator` resource. Run the following command:
```

> . *Technology Preview:* To enable global search to view your hub and managed cluster resources, add the `global-search-preview=true` annotation to the `search-v2-operator` resource. Run the following command:

Author: ```suggestion
. *Technology Preview:* Global search expands the search capabilities when you use {global-hub} to manage your environment. To enable global search, add the `global-search-preview=true` annotation to the `search-v2-operator` resource. Run the following command:
```

> . *Technology Preview:* Global search expands the search capabilities when you use {global-hub} to manage your environment. To enable global search, add the `global-search-preview=true` annotation to the `search-v2-operator` resource. Run the following command:

Reviewer 2: If `add the `global-search-preview=true` annotation to the `search-v2-operator` resource. Run the following command:` is all one command/action, I think it's fine to stay documented as it is. 

However, I do wonder if it's two distinct commands? So, something like the following: 

```suggestion
. *Technology Preview:* Global search expands the search capabilities when you use {global-hub} to manage your environment. To enable global search, complete the following steps:

1. Add the `global-search-preview=true` annotation to the `search-v2-operator` resource. 
2. Run the following command:
```

Reviewer 2: If it is all one action, perhaps it would be clearer if it's written like this:

```suggestion
. *Technology Preview:* Global search expands the search capabilities when you use {global-hub} to manage your environment. To enable global search, add the `global-search-preview=true` annotation to the `search-v2-operator` resource by running the following command:
```

> . *Technology Preview:* Global search expands the search capabilities when you use {global-hub} to manage your environment. To enable global search, add the `global-search-preview=true` annotation to the `search-v2-operator` resource. Run the following command:

Reviewer 2: I liked the "why" and "benefit" you had for your last doc in this PR. Would it be redundant if we included some of that content here? 

```suggestion
. *Technology Preview:* Global search expands the search capabilities when you use {global-hub} to manage your environment. This expanded search capability gives you more resources for you hub and managed clusters. To enable global search, add the `global-search-preview=true` annotation to the `search-v2-operator` resource. Run the following command:
```

+
[source,bash]
----

> oc annotate search search-v2-operator n opencluster-management global-search-preview=true

Reviewer: 2 missing dashes '-'
```suggestion
oc annotate search search-v2-operator -n open-cluster-management global-search-preview=true
```

----

+
The search operator is updated with the following status condition:

+
[source,yaml]
----
status:
  conditions:
     - lastTransitionTime: '2024-05-31T19:49:37Z'
     message: None
     reason: None
     status: 'True'

>      type: GlobalSearchReady

Reviewer: Fix indentation. Adding screen capture for reference. 
<img width="445" alt="image" src="https://github.com/stolostron/rhacm-docs/assets/4671325/d122b54d-49be-423d-af48-7728f2ee22f6">
```suggestion
status:
  conditions:
    - lastTransitionTime: '2024-05-31T19:49:37Z'
      message: None
      reason: None
      status: 'True'
      type: GlobalSearchReady
```

[#add-resources-search]
== Additional resources

> - See link:../global_hub/global_hub_overview.adoc#multicluster-global-hub[{global-hub} for more details.

Author: ```suggestion
- See link:../global_hub/global_hub_overview.adoc#multicluster-global-hub[{global-hub}] for more details.
```
